obitamockbank100.catalog.obita.tech/resources/developerGuide.md

```diff
@@ -0,0 +1,157 @@
+# Developer Guide: Obita Mock Banking
+
+## Introduction
+
+This developer guide is designed to help developers understand and work with the mock banking solution. This solution consists of creating bank accounts, making transactions, adding and managing beneficiaries, managing funds, managing standing orders, manage direct debits and third parties. Users can check their transaction history and query through all the transactions.
+
+## Solution Architecture
+
```

Comment:
Please use simple component diagram to explain the solution architecture of the sandbox core banking system. This diagram is there for open banking solutions